docs/IoT.js-API-UART.md

```diff
@@ -6,13 +6,14 @@ The following shows uart module APIs available for each platform.
 
 |  | Linux<br/>(Ubuntu) | Raspbian<br/>(Raspberry Pi) | Nuttx<br/>(STM32F4-Discovery) |
 | :---: | :---: | :---: | :---: |
-| uart.open | O | O | X |
-| uart.close | O | O | X |
-| uart.write | O | O | X |
+| uart.open | O | O | O |
+| uart.close | O | O | O |
+| uart.write | O | O | O |
 
```

Comment:
In first look, I thought it is strange since there is no read method. What about including 'data' event in this table?